pom.xml

```diff
@@ -1734,7 +1800,7 @@
             <dependency>
                 <groupId>org.apache.kafka</groupId>
                 <artifactId>kafka-clients</artifactId>
-                <version>2.4.1</version>
+                <version>${dep.confluent.version}-ccs</version>
```

Comment:
Can we use apache distribution of kafka libraries here and confluent? https://docs.confluent.io/platform/current/release-notes/index.html#cp-release-release-notes says you could use 3.3.2,